Client/Clique/Common/Models/CreateEventObject.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  createEventObject.swift
+//  Clique
+//
+//  Created by Infinum on 08.12.2022..
+//
+
+import Foundation
+
+struct CreateEventObject {
+    var eventName = ""
+    var eventLocation = ""
+    var eventTimeStampAPI = ""
+    var eventTimeStampPrint = ""
+    var participantsNumber = ""
+    var cost = ""
+    var currencyName = ""
+    var currencyId = ""
+    var categoryName = ""
+    var categoryId = ""
+    var description = ""
```

Comment:
Those values should not have a default value as `""`.
The reason for that is that, by default, this object has all values but is not valid at all.

Also, are all values `String`s and do you need all those values? E.g. timestamp should not be a `String`, but rather `Int`.

E.g. if you have `currencyId`, do you need `currencyName`? Based on the id, you can get the name, or even create a `Currency` object that holds that value. Another way would be to create an enum with defined id for currency; e.g.

```swift
enum Currency: String {
    case eur = "EUR"
    case hrk = "HRK"
    case usd = "USD" 
}
```
The same goes for the category.

Edit: I've seen you have a `Currency` model, you can basically load all available currencies in the flow and set the one here instead of setting the values from the client.